addons/editor-number-arrow-keys/userscript.js

```diff
@@ -87,7 +87,11 @@ export default async function ({ addon }) {
 
   document.body.addEventListener("keydown", (e) => {
     if (addon.self.disabled) return;
-    if (!e.target.classList.contains("blocklyHtmlInput")) return;
+    if (
+      !e.target.classList.contains("blocklyHtmlInput") &&
+      !e.target.classList.contains(addon.tab.scratchClass("input_input-form"))
```

Comment:
```suggestion
      !e.target.className.includes("input_input-form_"))
```

Reply:
(please test my suggestion to make sure it works)

Reply:
But why not just go with the more safe option?

Reply:
There are two different `input_input-form` classes - one from scratch-gui and one from scratch-paint. `scratchClass` will return the scratch-gui one.

Reply:
> There are two different `input_input-form` classes

TBH, I didn't know. But it's always a bad idea to use scratchClass inside a querySelector.